source/documentation/adrs/adr-013.html.erb.md

```diff
@@ -0,0 +1,43 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: ADR 013 Archiving the DNS repo
+last_reviewed_on: 2023-11-13
+review_in: 6 months
+---
+
+# ADR-013: Archiving the DNS repo
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The [DNS repo](https://github.com/ministryofjustice/dns) has historically held the following:
+
+- A dump of the dsd.io and justice.gov.uk hosted zones in Route53
+- Tooling to register domains on bulk
+- Tooling to maintain defensive domains
+
+The repository has not been in use for a long time now and is very out of date.
```

Comment:
Is it useful to say how long roughly? Eg "for 6-9 months" or since <date>.

Reply:
Will add it!